libs/ledgerjs/packages/devices/src/hid-framing.test.ts

```diff
@@ -0,0 +1,148 @@
+import createHIDframing, { ResponseAcc } from "./hid-framing";
+
+describe("createHIDframing.makeBlocks", () => {
+  const channel = Math.floor(Math.random() * 0xffff);
```

Comment:
what's the point of adding some randomness here?
couldn't we just use one arbitrary value?

Reply:
Just to show that it should work with whatever value for the channel (as this `channel` value is also randomly generated when we instantiate a USB HID Transport (for ex: in [TransportNodeHidNoEvents](https://github.com/LedgerHQ/ledger-live/blob/aa27e0f1e8043d0814d0f548cf52fb6d872a48ae/libs/ledgerjs/packages/hw-transport-node-hid-noevents/src/TransportNodeHid.ts#L81) )